resources/lang/en/admin/settings/general.php

```diff
@@ -47,6 +46,7 @@
     'barcode_type'				=> '2D Barcode Type',
     'alt_barcode_type'			=> '1D barcode type',
     'email_logo_size'       => 'Square logos in email look best. ',
+    'enabled'                   => 'Enabled',
```

Comment:
This feels like a more generic term than should go into the admin settings lang file, but the language organization is a mess anyway, so this is fine for now. Whatever solution we use to consolidate all of these strings will scoop that up as well.

In the future though, the UI pretty consistently uses the "Blah enabled" as the label and "Yes" for the checkbox. Even if that's not the "right" way to do it, we should consider being consistent throughout.